x-pack/plugins/security_solution/public/common/components/charts/barchart.tsx

```diff
@@ -51,11 +51,13 @@ const checkIfAnyValidSeriesExist = (
 export const BarChartBaseComponent = ({
   data,
   forceHiddenLegend = false,
+  yAxisTitle,
   ...chartConfigs
 }: {
   data: ChartSeriesData[];
   width: string | null | undefined;
   height: string | null | undefined;
+  yAxisTitle?: string | undefined;
```

Comment:
This should not affect any component as it is optional. When undefined, no y axis title is shown.